manifests/plugin/gutterball/config.pp

```diff
@@ -0,0 +1,14 @@
+# gutterball config
+class katello::plugin::gutterball::config(
+    $foreman_user  = 'foreman',
+    $foreman_group = 'foreman',
+    $foreman_plugins_dir = '/etc/foreman/plugins',
```

Comment:
@ekohl how many dirs are there for plugin configs? what is the correct one to use?